ilastik/applets/thresholdTwoLevels/thresholdTwoLevelsGui.py

```diff
@@ -431,4 +432,24 @@ def setupLayers(self):
             rawLayer.opacity = 1.0
             layers.append(rawLayer)
 
+            def toggleTopToBottom():
```

Comment:
This could be a method of the `LayerStackModel` class:

```python
def toggleTopToBottom(target: Union[int, Layer]) -> None:
    if isinstance(target, Layer):
        target = self.layerIndex(target)
    ...
```

Reply:
thx @emilmelnikov, I don't get sweaty hands with this refactor. We can address a more general solution (maybe also a la `LabelingGui` which adds raw layer automatically and assigns the shortcut) in a dedicated subclass of `LayerViewerGui`  as @btbest suggested, in the next release cycle.